### ISSUE TITLE
[BUG] [Homepage] Element must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by kstala** [shopware/frontends#1361]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

```
Element Location:
div[index="7"] > .sw-product-card.max-w-full.max-w-sm > .h-80.hover\:opacity-75.overflow-hidden > .top-5.-left-1.z-10 > .bg-\[\#FFBD5D\].px-2\.5.py-1\.5
<span class="bg-[#FFBD5D] px-2.5 py-1.5 color-white text-xl">Tip</span>
```
![Image](https://github.com/user-attachments/assets/c26c6c43-b09e-4c59-acf8-296d7683ae42)

To solve this problem, you need to fix the following:
Element has insufficient color contrast of 1.65 (foreground color: #ffffff, background color: #ffbd5d, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

### Environment

- OS:
- Node:
- pnpm:

**Comment by mdanilowicz** [comment]
Done in https://github.com/shopware/frontends/pull/1395/files
